px-context-browser.html

```diff
@@ -247,6 +275,16 @@ <h1 class="epsilon caps weight--normal u-m0" on-click="toggleColumnBrowser">
           value: function() {return {};}
         },
         /**
+         * If we have a selected multiple Items, it's stored here.
```

Comment:
Thanks for your valuable comments/suggestion. I really appreciate your detailed code review.
I'll add more comments as needed